lib/good_bread/accounts/user.ex

```diff
@@ -0,0 +1,20 @@
+defmodule GoodBread.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "users" do
+    field :name, :string
+    field :email, :string
+    field :address, :string
```

Comment:
Haven't decided whether or not to use oauth or just an email/password so left it out of this iteration